.eslintrc

```diff
@@ -24,13 +25,14 @@
     "rules": {
         // indentation should be 4 spaces (use soft tabs) and switch cases should be indented
         "indent": [2, 4, {
-            "SwitchCase": 1
+            "SwitchCase": 1,
+            "ignoredNodes": ["JSXElement *", "JSXElement"]
         }],
-        // require 4 spaces in JSX as well
-        "react/jsx-indent": [2, 4],
-        "react/jsx-indent-props": [2, 4],
+        "react/jsx-indent": [0],
```

Comment:
We should talk as a team about whether we want to adopt these eslint changes because they're going to affect the whole team on a daily basis.

Reply:
so these rules were actually conflicting with the indent rule above once we switched over to @babel/eslint-parser, you would have eslint --fix the spacing and then the opposite set of rules would complain and vice versa if you would do it again

so it was a neverending loop of fixing spacing but yeah i can better explain more what i did